crates/optimism/txpool/src/transaction.rs

```diff
@@ -244,6 +244,17 @@ where
     }
 }
 
+/// Helper trait to provide payload builder with access to conditionals and encoded bytes of
+/// transaction.
+pub trait OpPooledTx: MaybeConditionalTransaction + PoolTransaction {}
```

Comment:
out of scope, but the `Maybe` prefix on traits has been used to feature gate impl of a trait, so seems misused for `MaybeConditionalTransaction`